### ISSUE TITLE
type safety when instantiating the model

### ISSUE THREAD

**Issue by dengue8830** [RobinBuschmann/sequelize-typescript#286]
Its possible to maintain the type-safety when i create an instance of the model?

using the .create method

```
MyModel.create({ anAttrThatDontExistsInModel: 'something'}); // <- its fine for typescript, should throw an error
MyModel.create<MyModel>({ anAttrThatDontExistsInModel: 'something'}); // <- its fine for typescript, should throw an error
```

the same with .build, .bulkBuild, .bulkCreate

And with this way typescript throws an error:
`const model: MyModel = { anAttrThatExistsInModel: 1};`

> Property '$add' is missing in type '{ anAttrThatExistsInModel: number; }'.

The only way that i found to maintain type-safety is with

```
const model = new MyModel();
model.anAttrThatExistsInModel = 1; // ok for typescript
model.anAttrThatDontExistsInModel = 'something'; // typescript error, its ok
```

**Comment by RobinBuschmann** [comment]
Hey @dengue8830 you can either use the second generic parameter like

```ts
interface MyModelAttributes {
  someAttr: string;
}

await MyModel.create<MyModel, MyModelAttributes>({someAttr: 'hello'});
```
or update to the latest beta version via `npm install sequelize-typescript@next` and it should work without passing any generic type (`MyModel.create({...})`)

**Comment by dengue8830** [comment]
cool, thanks.

sorry for the delay, i didn't see the email
